hubmap/frontend/src/components/Experiments.js

```diff
@@ -75,7 +75,6 @@ class Experiments extends React.Component {
                   <StyledTableCell align="right">{experiment.image_count ? experiment._image_count : null}</StyledTableCell>
                 </StyledTableRow>
               ))
-
```

Comment:
Unless this was required by linting for some reason, we want to avoid edits to unrelated files during a PR. It makes it a bit harder to understand the history of a project.

Reply:
this is eslint. I am confident you are not reading my responses for your comments. I know this GUI is messy : when a reviewer starts a review, we receive single email, but when a developer responds to review comment, it goes as single email per comment.